hkmc2/shared/src/test/mlscript/handlers/NonLocalReturns.mls

```diff
@@ -82,3 +82,41 @@ fun f() =
     print("Bad")
 f()
 //│ = 100
+
+// broken due to handler does not handle object properly
```

Comment:
BTW do you know why it works with a class but not with an object?

Reply:
object instantiation emitted in JSBuilder is not checked for effect.

Reply:
Why not? Would it be hard to fix?

Reply:
Right, I just remembered there's already a corresponding item in https://github.com/hkust-taco/mlscript/issues/263